components/mpas-ocean/src/analysis_members/Registry_time_series_stats_climatology.xml

```diff
@@ -8,7 +8,7 @@
 		<nml_option name="config_AM_timeSeriesStatsClimatology_compute_on_startup"
 			type="logical"
 			default_value=".false."
-			description="Logical flag determining if an analysis member computation occurs on start-up. You likely want this off for this (time series) analysis member because it will accumulate any state prior to time stepping (double counting the last time step)."
+			description="Logical flag determining if an analysis member computation occurs on start-up. This should be set to .false. for better efficiency in most cases because the results of this first call will simply be thrown away.  The exception is at the beginning of new E3SM simulations (CONTINUE_RUN = FALSE), where the first time step will be missing if this is not set to .true."
```

Comment:
This and related descriptions for other time-series stats have been updated for better accuracy and clarity.  @jonbob, I didn't try to update the associate files in `bld` because I don't remember the exact process.  I can look that up, though, if you would like me to take care of it.